framework/src/org/checkerframework/framework/stub/StubParser.java

```diff
@@ -105,6 +106,9 @@
     /** A list of imports that are not annotation types. Used for importing enums. */
     private final List<String> imports;
 
+    /** A map of imported types names and types in a stub file. */
+    private final Map<String, TypeElement> importedTypes;
```

Comment:
Can you explain what the difference is between `imports` and `importedTypes`? Aren't the elements in the former the keys in the latter? What is different about importing enums vs. class literals? Do we need both?
Why store the `TypeElement` as value, when in the usage of the map you always call `asType()`? Is it to safe the `asType` calls that might never be needed? That is, you probably put more things into the map than will ever be used as class literals later on. So it's easier to store the TypeElements instead of the result of the Types from the `asType()` calls.